benchmarks/frameworks/torch/mpc/scripts/abstract/abstract_benchmark.py

```diff
@@ -0,0 +1,40 @@
+import abc
+
+from .workers_initialization import workers, hook
+
+
+class AbstractBenchmark(metaclass=abc.ABCMeta):
```

Comment:
Q: Can you use ```class AbstractBenchmark(ABC) and do ```from abc import ABC```. There are a few examples in ```syft/generic/abstract```

Reply:
Sure!